src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import { FormEvent, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [prevTodos, setTodos] = useState(todos);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleCheck, setTitleCheck] = useState(false);
+  const [userIdCheck, setuserIdCheck] = useState(false);
+
+  const createTodo = () => {
+    return (
+      {
+        id: Math.max(0, ...prevTodos.map(({ id }) => id + 1)),
+        userId,
+        title,
+        completed: false,
+        user: getUser(userId) || null,
```

Comment:
getUser returns null if user doesn’t exist 
```suggestion
        user: getUser(userId),
```